docker/puppetserver-standalone/spec/puppetserver_spec.rb

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env ruby
+
+require 'rspec/core'
+
+describe 'puppetserver-standalone container' do
+  def puppetserver_health_check(container)
+    status = %x(docker inspect "#{container}" --format '{{.State.Health.Status}}').chomp
+    STDOUT.puts "queried health status of #{container}: #{status}"
+    return status
+  end
+
+  before(:all) do
+    %x(docker pull puppet/puppet-agent-alpine:latest)
+    @image = ENV['PUPPET_TEST_DOCKER_IMAGE']
+    if @image.nil?
+      error_message = <<-MSG
+* * * * *
+  PUPPET_TEST_DOCKER_IMAGE environment variable must be set so we
+  know which image to test against!
+* * * * *
+      MSG
+      fail error_message
+    end
+
+    @container = %x(docker run --rm --detach --env DNS_ALT_NAMES=puppet --name puppet.test --hostname puppet.test --publish 8140:8140 #{@image}).chomp
```

Comment:
Should we randomize the 8140 locally?

Reply:
Actually, since we're linking the agent to the master container I think we can get away without publishing the ports ... testing!

Reply:
Yup, looks like that works \o/